scripts/ci/libraries/_initialization.sh

```diff
@@ -148,6 +151,23 @@ function initialize_common_environment {
         )
     fi
 
+    # Common variables passed via docker commands
+    EXTRA_DOCKER_FLAGS+=(
+        "--env" "INSTALL_AIRFLOW_VERSION"
+        "--env" "PYTHONDONTWRITEBYTECODE"
+        "--env" "VERBOSE"
+        "--env" "VERBOSE_COMMANDS"
+        "--env" "HOST_USER_ID"
+        "--env" "HOST_GROUP_ID"
+        "--env" "HOST_OS"
+        "--env" "HOST_HOME"
+        "--env" "HOST_AIRFLOW_SOURCES"
+        "--env" "PYTHON_MAJOR_MINOR_VERSION"
+        "--env" "VERSION_SUFFIX_FOR_PYPI"
+        "--env" "VERSION_SUFFIX_FOR_SVN"
+        "--env" "CI"
```

Comment:
How about using [--env-file](https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables--e---env---env-file) ?

Reply:
I was under the impression that env-file requires var=value but I looked it up and we could indeed change it this way, Let me see if I can make it nicely :)

Reply:
I did it and made it even nicer :).